pkg/status/status.go

```diff
@@ -324,10 +319,6 @@ func (s *status) syncChannelReceiver() {
 // monitorClusterStatus updates the ClusterOperator's status based on
 // the number of successful syncs / total syncs
 func (s *status) monitorClusterStatus() {
-	// Signal to the main channel that we have stopped reporting status.
-	defer func() {
-		close(s.monitorDoneCh)
-	}()
 	for {
 		select {
 		case <-s.stopCh:
```

Comment:
Is the comment below accurate?

Reply:
It is not, this is addressed in the latest commit.